### ISSUE TITLE
Support for links on creation and edition

### ISSUE THREAD

**Issue by cavalle** [bebanjo/almodovar#3]
**NOTE:** This one should probably be done after #2

Currently there's no documented (nor implemented, I believe) way of setting the href of a link on creation and modification of a resource. This should support both links that are resources of the same api and links to external resources.

A nice to have features would be to support referencing other resources. Example:

``` ruby
title = Almodovar::Resource('https://api.bebanjo.net/titles/1')
series = Almodovar::Resource('https://api.bebanjo.net/series/1')

title.update(:title => {:series_link => series })
# Previous link and the next one would have the same effect
title.update(:title => {:series_link => 'https://api.bebanjo.net/series/1'})
```

The `series_link` is a suggestion this is not yet decided
